package.json

```diff
@@ -18,7 +18,8 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "xo && node test.js"
+    "test": "xo && nyc node test.js",
+    "coverage": "xo && nyc --reporter=text-lcov node test.js | coveralls"
```

Comment:
I didn't realize xo was the linter, thought it was part of running the test suite, fixed.